frontend/talentsearch/resources/js/components/experienceAccordion/ExperienceAccordion.tsx

```diff
@@ -0,0 +1,175 @@
+    // community
+    project?: string;
+    // education
+    type?: string;
+    status?: string;
+    thesisTitle?: string;
+    // personal
+    description?: string;
+    // work
+    division?: string;
+    // linked skills
+    experienceSkills?: [];
+  };
+}
+
+const ExperienceAccordion: React.FunctionComponent<AccordionProps> = ({
+  anExperience,
+}) => {
+  // destructuring required due to linting?
+  const {
+    experienceType,
+    title,
+    awardedDate,
+    issuedBy,
+    role,
+    organization,
+    areaStudy,
+    institution,
+    startDate,
+    endDate,
+    awardedTo,
+    awardedScope,
+    project,
+    type,
+    status,
+    thesisTitle,
+    description,
+    division,
+  } = anExperience;
+
+  // experience type is required with 5 possibilities, build different accordion around which type it is
+  let experienceInstance;
+  if (experienceType === "AwardExperience") {
+    experienceInstance = (
+      <Accordion
+        title={`${title || ""} - ${issuedBy || ""}`}
+        subtitle={`Since: ${awardedDate || ""}`}
+        context="skills count"
+        Icon={BriefCaseIcon}
+      >
+        <div>
+          <p>
+            {title} issued by {issuedBy}
+          </p>
+          <p>Awarded to: {awardedTo}</p>
+          <p>Scope: {awardedScope}</p>
+        </div>
+      </Accordion>
+    );
+  } else if (experienceType === "CommunityExperience") {
+    experienceInstance = (
+      <Accordion
+        title={`${role || ""} at ${organization || ""}`}
+        subtitle={
+          endDate
+            ? `${startDate || ""} - ${endDate || ""}`
+            : `Since: ${startDate || ""}`
+        }
+        context="skills count"
+        Icon={BriefCaseIcon}
+      >
+        {" "}
+        <div>
+          <p>
+            {role} at {organization}
+          </p>
+          <p>{project}</p>
+        </div>
+      </Accordion>
+    );
+  } else if (experienceType === "EducationExperience") {
+    experienceInstance = (
+      <Accordion
+        title={`${areaStudy || ""} at ${institution || ""}`}
+        subtitle={
+          endDate
+            ? `${startDate || ""} - ${endDate || ""}`
+            : `Since: ${startDate || ""}`
+        }
+        context="skills count"
+        Icon={BriefCaseIcon}
+      >
+        <div>
+          <p>
+            {type} {status}
+          </p>
+          <p>
+            {areaStudy} at {institution}
+          </p>
+          <p>{thesisTitle ? `Thesis: ${thesisTitle}` : ""}</p>
+        </div>
+      </Accordion>
+    );
+  } else if (experienceType === "PersonalExperience") {
+    experienceInstance = (
+      <Accordion
+        title={title || ""}
+        subtitle={
+          endDate
+            ? `${startDate || ""} - ${endDate || ""}`
+            : `Since: ${startDate || ""}`
+        }
+        context="skills count"
+        Icon={BriefCaseIcon}
+      >
+        <div>
+          <p>{description}</p>
+        </div>
+      </Accordion>
+    );
+  } else {
```

Comment:
I suggest still testing for work experience here and letting your `else` be some kind of _unknown experience_ block.  This will make it easier to troubleshoot if we add another experience type but forget to update this component.

Reply:
definitely will do